scss/_color-filters.scss

```diff
@@ -0,0 +1,142 @@
+// MAIN COLORS
+.gor-color--gorila-primary {
```

Comment:
Isn't it better to add something to be easier to identify that these classes apply filters? 🤔

Reply:
Could be done, but I'm afraid of including something else and being stuck with names that are too long... Any suggestions? 🤔

Reply:
`.gor-filter` isn't a bad idea but it might conflict with some component... we may try to add other selector (maybe `img`) to avoid this conflict...🤔
Is it required to have the `gor` prefix? If not, maybe we can try something like `icon-filter`...

Reply:
The prefix isn't required, but helps avoiding conflicts. `icon-filter` could work, but at the general case these classes could be applied to anything so I'm hesitant in including the name of any specific element.... 🤔

Reply:
Maybe `color-filter`? 🤔